src/lashup_kv.erl

```diff
@@ -322,10 +322,14 @@ prepare_kv(Key, Map0, VClock0, Op) ->
   VClock1 = riak_dt_vclock:increment(Node, VClock0),
   Counter = riak_dt_vclock:get_counter(Node, VClock1),
   Dot = {Node, Counter},
-  {ok, Map1} = riak_dt_map:update(Op, Dot, Map0),
+  Map2 =
+    case riak_dt_map:update(Op, Dot, Map0) of
```

Comment:
@GoelDeepak JFYI I've found the reason of yesterday's dialyzer issue in riak_dt, fixed it [there](https://github.com/dcos/riak_dt/commit/d29fd7fb6135d460621aba69aac15fec59838167) and removed `apply/3` here.